### DISCUSSION TITLE
Lab 3.01 Task 2

### DISCUSSION THREAD

**Discussion by JuanCH94** [rramosp/ai4eng.v1#111] [03 - GENERACION DE MODELOS]
¡Buenas tardes¡  

Espero me puedan ayudar, el código de la tarea 2 de este laboratorio me corre bien, pero al realizar en envió para calificar no esta dando lo esperado, que puedo tener malo. 

![3 01 taks II](https://user-images.githubusercontent.com/98975811/152398566-d82ae4fc-d673-409b-a054-946781e847da.PNG)
![Resultado Task 2](https://user-images.githubusercontent.com/98975811/152398569-b221f465-7638-4e3d-ae8a-fd133c0c3bef.PNG)

**Comment by Felipe-RA** [comment]
Saludos JuanCH94

Disculpa la tardanza en responder, no habia visto tu post.

EN las capturas que enviaste el tooltip de ndarray no deja ver bien tu código.

Por favor vuelvelo a poner, pero en lugar de como imágen, como bloque de código
